packages/main/src/Carousel.js

```diff
@@ -0,0 +1,296 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import {
+	isLeft,
+	isRight,
+	isDown,
+	isUp,
+} from "@ui5/webcomponents-base/dist/events/PseudoEvents.js";
+import {
+	fetchI18nBundle,
+	getI18nBundle,
+} from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import ScrollEnablement from "@ui5/webcomponents-base/dist/delegate/ScrollEnablement.js";
+import { isDesktop } from "@ui5/webcomponents-base/dist/Device.js";
+import {
+	CAROUSEL_OF_TEXT,
+} from "./generated/i18n/i18n-defaults.js";
+import CarouselArrowsPlacement from "./types/CarouselArrowsPlacement.js";
+import CarouselTemplate from "./generated/templates/CarouselTemplate.lit.js";
+
+// Styles
+import CarouselCss from "./generated/themes/Carousel.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-carousel",
+	properties: /** @lends sap.ui.webcomponents.main.Carousel.prototype */ {
+		/**
+		 * Defines whether the carousel should loop, i.e show the first page after the last page is reached and vice versa.
+		 * @type {Boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		cycling: {
+			type: Boolean,
+		},
+
+		/**
+		 * Sets the amount of items per page. If this property is set, on mobile devices it will always fallback to 1.
+		 * @type {Integer}
+		 * @defaultvalue 1
+		 * @public
+		 */
+		itemsPerPage: {
+			type: Integer,
+			defaultValue: 1,
+		},
+
+		/**
+		 * If set to true the navigation is hidden.
+		 * @type {Boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		hideNavigation: {
+			type: Boolean,
+		},
+
+		/**
+		 * Defines the index of the initially selected slot.
```

Comment:
We better add a note, that if itemsPerPage is set, it's the index of the page, rather than the individual item/slot